src/u_mapinfo.c

```diff
@@ -667,6 +668,15 @@ static int ParseStandardProperty(u_scanner_t* s, mapentry_t *mape)
     U_GetNextToken(s, true);
   } while (U_CheckToken(s, ','));
 
+  if (!mape->nextmap[0] && !mape->endpic[0])
+  {
+    int ep, map;
+
+    G_ValidateMapName(mape->mapname, &ep, &map);
+
+    strcpy(mape->nextmap, MAPNAME(ep, map + 1));
```

Comment:
This is the same logic as pr+, which is good, but looking at it I release this makes the default progression map 32 -> map 33 🤦 
Just confirmed in dsda-doom you crash when leaving map 32 if there's a map 32 entry without next defined...

Reply:
It's probably best to just leave it as is I guess. There isn't exactly a "normal" progression of maps in umapinfo, so maybe this is intentional.